docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,370 @@
+---
+title: Generating Short-Lived Host Certificates with Machine ID
+description: Issuing Host Certificates with Machine ID for OpenSSH servers.
+---
+
+Host certificates are generally created on Teleport for access to OpenSSH servers that can not otherwise join a Teleport cluster.
+While long-lived certificates may be applied for this purpose, short-lived and regularly rotated host certificates generated
+through Teleport help to provide a number of security benefits over their long-lived counterparts. Regular certificate rotation reduces risk by ensuring that any potentially stolen
+certificates are usable for a shorter period of time. Additionally, when coupled with Teleport's [RBAC](https://goteleport.com/docs/access-controls/guides/role-templates/)
+support and host certificate **Principals** and **Predicates**, you can apply limitations to both the process of creating host certificates,
+as well as the host certificates themselves.
+
+In this guide, you will learn how to create host certificates through Machine ID for OpenSSH servers, allowing for the benefits of host certificates with OpenSSH nodes,
+and reducing risk by ensuring that short-lived certificates adhering to the principle of least privelege can be successfully applied to Teleport clusters.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!/docs/pages/includes/tctl.mdx!)
+
+- OpenSSH server `sshd` version 6.9 or above installed. The SSH port on this host must be open to traffic from the Teleport Proxy Service host.
+
+  ```code
+  $ ssh -V
+  ```
+
+- A Linux based host that can support Machine ID. This "Machine ID Host" is the host that will be used to create Host Certificates.
```

Comment:
Hm, I'd expect that this "Machine ID host" to be the same machine as the one with OpenSSH running on it.